docs/usage.md

```diff
@@ -189,9 +183,13 @@ Nextflow
 nextflow run sanger-tol/blobtoolkit --input SAMPLESHEET --fasta GENOME –-accession GCA_ACCESSION --taxon TAXON_ID --taxdump TAXDUMP_DB --blastp DMND_db --blastn BLASTN_DB --blastx BLASTX_DB
 
 # Draft Assemblies
-nextflow run sanger-tol/blobtoolkit --input SAMPLESHEET --fasta GENOME –-accession TAG --taxon TAXON_ID --yaml CONFIG --taxdump TAXDUMP_DB --blastp DMND_db --blastn BLASTN_DB --blastx BLASTX_DB
+nextflow run sanger-tol/blobtoolkit --input SAMPLESHEET --fasta GENOME --taxon TAXON_ID --taxdump TAXDUMP_DB --blastp DMND_db --blastn BLASTN_DB --blastx BLASTX_DB
```

Comment:
It looks like the taxID given by the user is used as the argument for `-negative_taxids` in the downstream `blastn` command. I assume the purpose of `-negative_taxids` is to ignore hits to the same assembly itself if the assembly is already in the `nt` database. I think this makes sense to do when running BlobToolKit with already published assemblies but not when running it with draft assemblies.

When testing `sanger-tol/blobtoolkit` I ran into a `blastn` crash that seems to originate from the `-negative_taxids` part in the BLAST command: 
```
BLAST Database error: Taxonomy ID(s) not found.Taxonomy ID(s) not found. This could be because the ID(s) provided are not at or below the species 
level. Please use get_species_taxids.sh to get taxids for nodes higher than species (see https://www.ncbi.nlm.nih.gov/books/NBK546209/).
```
The taxID that I used was 73239. This actually already is at subspecies level but it somehow still triggered this error. I think this crash wouldn't have happened if  the run with the draft assembly hadn't required the taxID for the `-negative_taxids` bit. So I think it would be better to remove the `-negative_taxids` part when the run is done with a draft assembly

Reply:
For comparison, can you share an example run of the Snakemake pipeline ? I was under the impression that `-negative_taxids` was _always_ used.

This "BLAST Database error: Taxonomy ID(s) not found." error has been fixed in #103, which is included in this branch since my last rebase 2 weeks ago. Are you still seeing the error on the latest version of the branch ?

Reply:
About the `blastn` crash: yes, sorry, turns out I was accidentally running out of date code when I had this crash yesterday. The crash didn't happen when I retried the run with updated repository copy.